src/main/java/uk/gov/companieshouse/uri/web/model/CompanyDetails.java

```diff
@@ -164,6 +164,26 @@ public MortgageTotals getMortgageTotals() {
     public void setMortgageTotals(MortgageTotals mortgageTotals) {
         this.mortgageTotals = mortgageTotals;
     }
+    
+    /**
+     * Converts a string date of format 'DD/MM/YYYY' to 'YYYY-MM-DD'
```

Comment:
what about American dates with month first? Can we even tell?

Reply:
All dates are UK style, and a fixed format, as they have been set from a LocalDate object earlier in the code.